_includes/datagov-search-display.html

```diff
@@ -69,10 +69,15 @@
   {{- content -}}
 </div> 
 <div class="search pagination padding--bottom--xl">
-  <span class="sgds-icon sgds-icon-arrow-left is-size-4"></span>
+  <span class="bx bx-chevrons-left is-size-4"></span>
+  <div class="px-1 is-hidden-mobile"></div>
+  <span class="sgds-icon sgds-icon-arrow-left is-size-4 is-hidden-desktop is-hidden-tablet-only"></span>
   {%- comment -%} To insert page selectors {%- endcomment -%}
-  <div id="paginator-pages"></div>
-  <span class="sgds-icon sgds-icon-arrow-right is-size-4"></span>
+  <div id="paginator-pages" class="is-hidden-mobile"></div>
+  <div id="paginator-pages-mobile" class="is-hidden-desktop is-hidden-tablet-only is-full-width"></div>
+  <span class="sgds-icon sgds-icon-arrow-right is-size-4 is-hidden-desktop is-hidden-tablet-only"></span>
+  <div class="px-1 is-hidden-mobile"></div>
+  <span class="bx bx-chevrons-right is-size-4"></span>
```

Comment:
nit: can we add aria labels here since we do not have text here for screen readers?

Reply:
wah +1 quick win

Reply:
https://github.com/isomerpages/isomerpages-template/pull/320/commits/6a4df4a5910c146b4b01463a583e6341af5373ba